jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/BPMN2DataServicesReferencesTest.java

```diff
@@ -510,7 +521,41 @@ public void testSignalsAndGlobals() throws IOException {
                     "person".equals(globalName) || "name".equals(globalName) );
         }
 
+        assertNotNull("Signals description should not be null", procDef.getSignalsDesc());
+        assertNotNull("Messages description should not be null", procDef.getMessagesDesc());
+        assertTrue("Signal description should be empty", procDef.getSignalsDesc().isEmpty());
+        assertTrue("Messages description should be empty", procDef.getMessagesDesc().isEmpty());
+
         // cleanup
         processService.abortProcessInstance(procInstId);
     }
+
+    @Test
+    public void testSignalsDesc() throws IOException {
+        ProcessDefinition procDef = bpmn2Service.getProcessDefinition(deploymentId, "org.jbpm.boundarysignal");
+        assertNotNull(procDef);
+
+        Collection<SignalDesc> signals = procDef.getSignalsDesc();
+        assertNotNull("Signals description should not be null", signals);
+        assertFalse("Signal description should be empty", signals.isEmpty());
```

Comment:
Should be changed to:
```suggestion
        assertFalse("Signal description should not be empty", signals.isEmpty());
```